coordinator/Services/JudgerService.cs

```diff
@@ -44,5 +47,46 @@ public class JudgerService {
                 .SingleOrDefaultAsync();
             return judger;
         }
+
+        public async Task<List<JudgerEntry>> QueryJudger(
+            List<string> tag, string fromId = "", int count = 50) {
+            var judger = await db.Judgers
+                .Where(j => j.Tags != null && j.Tags.All(t => tag.Contains(t)))
```

Comment:
Docs: maybe explain what each filter is doing?

Reply:
I thought it was clear enough...